src/pages/Home/subcomponents/FriendReadonlyCountry.js

```diff
@@ -14,8 +14,9 @@ import FriendClickedCountryBlank from "../../../components/Prompts/FriendClicked
 import MapScorecard from "./MapScorecard";
 import MapInfoContainer from "./MapInfoContainer";
 import MapChangeIcon from "../../../icons/MapChangeIcon";
+import ReadonlySignupPrompt from "../../../components/Prompts/ReadonlySignupPrompt";
 
-const FriendReadonlyCountry = props => {
+const FriendReadonlyCountry = (props) => {
```

Comment:
@GreenWithMV I'm pretty sure the preferred way to do this is:

```react
const FriendReadonlyCountry = ({ props }) => {
```

This article touches on the topic briefly: https://www.robinwieruch.de/react-pass-props-to-component (can't link the anchors, but it's in the article around "One would call it React props destructuring:").

If it's not impacting anything right now, feel free to merge/ignore for now- but may be a good one to talk through in a future call!